lib/grizzly/inclusion_handler.ex

```diff
@@ -20,9 +20,14 @@ defmodule Grizzly.InclusionHandler do
 
   If you are not handling S2 devices the last two can be ignored until you are
   ready to provide support them.
+
+   The error-handling function will want to handle timeouts, particularly on state :dsk_requested,
+   if you are handling S2-authenticated devices.
+
   """
 
   alias Grizzly.ZWave.Command
 
   @callback handle_command(Command.t(), keyword()) :: :ok
+  @callback handle_error({:error, atom, atom}, keyword()) :: :ok
```

Comment:
Would you be open to making this `handle_timeout/1` where:

```
@callback handle_timeout(atom(), keyword()) :: :ok
```
If I understand correctly that is the only thing we mark as an error right now and since we know the concrete error it might nice to make handling this more explict?

The first argument ins the inclusion state. I have some thoughts on updating this a little to help expose that type to the public API, but I rather do that after this PR is done.

Reply:
Done.